lvm-storage/operator/base/namespace.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    openshift.io/display-name: OpenShift LVM Storage Operator
+  labels:
+    openshift.io/cluster-monitoring: 'true'
+  name: openshift-storage
```

Comment:
don't you want to target a different namespace?

Reply:
I just the deault ns in the package manifest and doc reference so to align:

```
$ tasty info lvms-operator
Providing information on operator lvms-operator
source:  redhat-operators
channels:  [stable-4.12]
defaultchannel:  stable-4.12
target namespace:  openshift-storage
csv:  lvms-operator.v4.12.1
description:  Logical volume manager storage provides dynamically provisioned local storage.

```